js/audioManager.js

```diff
@@ -135,15 +153,32 @@ export class AudioManager {
     }
   }
 
+  // Called after changing something in this.loopAudioBuffer
+  audioDataChanged() {
+    if (this._bufSource !== null) {
+      this._bufSource.stop();
+      this._bufSource.disconnect();
+    }
```

Comment:
Would be nice to skip the rest of this on chromium when this is called not for first time